src/main/java/ru/test/restservice/dto/FileItemDTO.java

```diff
@@ -1,15 +1,23 @@
 package ru.test.restservice.dto;
-
-import lombok.AllArgsConstructor;
 import lombok.Data;
 
+import java.beans.ConstructorProperties;
 import java.util.List;
 
 @Data
-@AllArgsConstructor
 public class FileItemDTO {
     public String name;
     public String type;
     public String path;
     public List<String> content;
+
+    // Встроенный Jackson не ест конструктор, сгенерированный @AllArgsConstructor
+    // На тестах ломалось и изменения в lombok.config сбрасываются при сборке проекта, так что добавил свой
+    @ConstructorProperties({"name", "type", "path", "content"})
+    public FileItemDTO(String name, String type, String path, List<String> content) {
+        this.name = name;
+        this.type = type;
+        this.path = path;
+        this.content = content;
+    }
```

Comment:
Jackson'у нужен дефолтный конструктор и setter'ы
```suggestion
@AllArgsConstuctor
@NoArgsConstructor
public class FileItemDTO {
    public String name;
    public String type;
    public String path;
    public List<String> content;
```

Reply:
Я для этого и написал комментарий. Не работают тесты с таким кодом

Reply:
слыш исправлять